### ISSUE TITLE
Species lookup failing on Mac systems?

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#1175]
I've only had this reported by one person, but the suggestion is that on our recording forms the species name autocomplete boxes are not functioning on Mac systems:

> I do all my work on a Mac. Up until mid Ocotober, species entry was not a problem but when I went to enter species in November I was unable to, and still cannot. There was an update on the species list??
> However, it works fine on a pc Windows.
> This suggests that the server for the list of species (which now does not appear), has become  incompatible with MacOS system and does not appear in the species box.
> As I use a Mac is it possible to enable the species server to recognse this?

Not sure if the move to Drupal 9 will have any effect on this. And I don't know if the issue is specific to this one user (I've not heard from any other Mac users).

**Comment by BirenRathod** [comment]
@kitenetter If I'm not wrong, This issue is related to SSL certificate. For some reason on Mac system, browser used old SSL certificate even though it has updated way before on the server. I'm checking this if that is the case.

**Comment by BirenRathod** [comment]
@kitenetter I checked on both servers iRecord & Indicia and found not certificate has been expired in October month, so nothing to relate this with SSL certificate.

Also, I checked on MacOS too and found no problem of getting species list. Although I only checked on enter-record-list page but again if list appear on that page means no problem  of getting species list from the server.

On Mac, I suggest he should do two things. 1. upgrade browser to latest version and 2. Clear cache from it.
